pyproject.toml

```diff
@@ -42,6 +42,7 @@ pyyaml = "*"
 starlette = "*"
 typing-extensions = "*"
 requests = { version = "*", optional = true }
+Brotli = { version = "*", optional = true }
```

Comment:
No need to capitalize the dependency name, pypi is case insensitive. Please keep consistent lower case.